MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -130,29 +124,6 @@ void Setup()
 			gd.DepthStencilState = _depthStencilState;
 			gd.RasterizerState = _rasterizerState;
 			gd.SamplerStates[0] = _samplerState;
-			
-			var vp = gd.Viewport;
```

Comment:
We should always apply the _matrixTransform on the _spriteEffect, see: https://github.com/MonoGame/MonoGame/pull/5244#issuecomment-256257160